src/cmake/ie_parallel.cmake

```diff
@@ -113,6 +117,7 @@ function(set_ie_threading_interface_for TARGET_NAME)
         if (TBB_FOUND)
             set(IE_THREAD_DEFINE "IE_THREAD_TBB")
             ie_target_link_libraries(${TARGET_NAME} ${LINK_TYPE} ${TBB_IMPORTED_TARGETS})
+            target_compile_definitions(${TARGET_NAME} ${COMPILE_DEF_TYPE} TBB_PREVIEW_WAITING_FOR_WORKERS=1)
```

Comment:
I suppose we should duplicate this to `ie_parallel.hpp` since if the customer does not use cmake, such definition is lost.

Reply:
ok, let me check how to solve it.

Reply:
just put definition to `ie_parallel.hpp` as well :)